### ISSUE TITLE
how to get the accuracy of scholar(covariates)

### ISSUE THREAD

**Issue by ItGirls** [dallascard/scholar#6]
Hi, could you tell me how to get the accuracy of your model scholar(covariates)? I ran you code in the help of your README file, but there are some problems.

I successfully got the accuracy of your model scholar(labels) by run`python run_scholar.py data/imdb/processed/ -k 50 --test-prefix test --labels sentiment --device 0`。However, I am confused by the scholar(covariates), I don't know how to get the accuracy for it.

Here, I've  tried two ways. By running `python3 run_scholar.py data/imdb/processed/ -k 50 --test-prefix test --topic-covars sentiment --labels sentiment --device 0`, I got nothing. By running `python3 run_scholar.py data/imdb/processed/ -k 50 --test-prefix test --labels sentiment --prior-covars sentiment --device 0`, I got the result, but I don't know if this is the right code to get accuracy of  model scholar(covariates). Therefor，could you tell me how to get the accuracy of your model scholar(covariates)?

Thank you！

**Comment by dallascard** [comment]
Sure, for that feature, you need to use the tensorflow version. The command would be:
``` python run_scholar_tf.py data/imdb/processed/ -k 50 --test-prefix test --topic-covars sentiment --infer-covars```

Unfortunately that feature was never ported over to the pytorch version.

**Comment by ItGirls** [comment]
> Sure, for that feature, you need to use the tensorflow version. The command would be:
> ` python run_scholar_tf.py data/imdb/processed/ -k 50 --test-prefix test --topic-covars sentiment --infer-covars`
> 
> Unfortunately that feature was never ported over to the pytorch version.
so, does it mean for the result of scholar(covariates), I don't need to change the process of the data, I just need replace the  command  (in pytorch version)  with the command  you gave above.

**Comment by dallascard** [comment]
Yes, the data processing is the same, but you would need to modify the code for the pytorch version in order to implement that feature (or you can use the tensorflow version).

**Comment by ItGirls** [comment]
> Yes, the data processing is the same, but you would need to modify the code for the pytorch version in order to implement that feature (or you can use the tensorflow version).

Thank you.
